insitupy/io/metadata.py

```diff
@@ -19,23 +18,23 @@ class MetaDataParser:
     Base class for parsing metadata
     """
     OUT_TIMEZONE = "UTC"
-    ID_NAMES = ["pitid", "pit_id"]
-    SITE_NAME_NAMES = ["location", "site_name"]
-    LAT_NAMES = ["lat", "latitude"]
-    LON_NAMES = ["lon", "lon", "longitude"]
-    UTM_EPSG_PREFIX = "269"
-    NORTHERN_HEMISPHERE = True
```

Comment:
Remove most of this and relied on the keys we are mapping it to with the YAML files